questions.md

```diff
@@ -0,0 +1,4 @@
+
+
+1) how does git "remember" current context ?
```

Comment:
Hi @smumerlin , when you refer to "context" I'm assuming you mean how does git know which branch you are working on. Remember that git has a pointer, HEAD, which keeps track of which branch (and commits) you are using in your working directory.